precise_description/robots/PF400_long.yaml

```diff
@@ -60,8 +60,8 @@ arm:
       yaw: 0.0
     limit:
       effort: 1000.0
-      lower: -16.9296937443  # -970
-      upper: +16.9296937443  # +970
+      lower: -6.28  # -970
+      upper: +6.28  # +970
```

Comment:
@benmaidel @destogl
I'd rather change this in the joint_limits.yaml of the kevin_moveit_config

Reply:
as soon as we get rid of PKP movements the limitation to +-2Pi is not needed anymore.

Reply:
@fmessmer I agree this is the right approach.

Reply:
these limits are enforced in https://github.com/mojin-robotics/kevin/pull/77/commits/f77e92c6f50f86b0c1d69609faf8b5760ccdda8f now and reverted here